src/main/java/dp/api/dataset/models/Metadata.java

```diff
@@ -231,7 +233,16 @@ public void setQmi(GeneralDetails qmi) {
     }
 
     public String getReleaseDate() {
-        return releaseDate;
+
+        if (this.releaseDate == null) {
```

Comment:
You can use `StringUtils.IsEmpty(...)` which will handle both `null` and `""` cases.